lib/msal-common/src/server/ServerAuthorizationTokenResponse.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ServerError } from "../error/ServerError";
+
+/**
+ * Deserialized response object from server authorization code request.
+ * - 
+ */
+export type ServerAuthorizationTokenResponse = {
+    // Success
+    token_type?: string;
```

Comment:
Should these be non-optional? Or at least one/some non-optional?

Reply:
Is there a way to have one or the other?

We will either get:
```
token_type: string;
scope: string;
expires_in: number;
ext_expires_in: number;
access_token: string;
refresh_token: string;
id_token: string;
```
in case of success, or 
```
error: string;
error_description: string;
error_codes: Array<string>;
timestamp: string;
trace_id: string;
correlation_id: string;
```
But I'm not sure how to have typescript set it to one or the other.

Reply:
I think you would have to have two separate types for that.

Reply:
Resolved offline.